gear/user_management/src/python/user_app/main.py

```diff
@@ -25,36 +44,34 @@ def create_user(proxy: FlywheelProxy, user_entry: UserDirectoryEntry) -> User:
     Returns:
       the ID for flywheel User created from the directory entry
     """
-    user_id = user_entry.credentials['id']
-    new_id = proxy.add_user(
-        User(id=user_id,
-             firstname=user_entry.name['first_name'],
-             lastname=user_entry.name['last_name'],
-             email=user_id))
-    user = proxy.find_user(new_id)
+    new_id = proxy.add_user(create_user(user_entry=user_entry))
+    user = proxy.find_user(user_entry.user_id)
```

Comment:
Do we need to check new_id and user_entry.user_id are the same?

Reply:
I don't think so. the User created by create_user has the id set to user_entry.user_id and there is nothing in proxy.add_user that should result in a different value. 

are you aware of anything that would indicate this is an issue?